extensions/sql-migration/src/wizard/integrationRuntimePage.ts

```diff
@@ -732,6 +731,11 @@ export class IntergrationRuntimePage extends MigrationWizardPage {
 	}
 
 	private atleastOneNodeOnline(nodes: IntegrationRuntimeNode[]): boolean {
-		return nodes.some(node => node.status === constants.ONLINE);
```

Comment:
did we get any issue from this line, these two codes should work adjunctly same.

Reply:
reverted back, miss in merge conflict